### PR TITLE
konvoy: note about Kubernetes version for AKS

### PR DESCRIPTION
**Pull request by dkoshkin**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-83927

## Description of changes being made
* Documented the Kubernetes version to use for AKS. Also added a note about the available versions.
image

* Also removed the "Replace Node" page, this is not correct for AKS and because it uses a MachinePool and not a MachinedeDeployment its not possible to target a machine to delete.

## Checklist

- [x] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [x] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by gflorinskaya — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-new/index.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Before you start, make sure you have completed the steps in [Bootstrap][bootstra
     azuremanagedmachinepool.infrastructure.cluster.x-k8s.io/mpd56zd created
     ```
 
+   <p class="message--important"><strong>IMPORTANT: </strong>Not all released versions of Kubernetes are available in Azure when creating a Managed Clusters (AKS). Use the Azure portal or CLI to <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions">find the versions supported in your cluster's location</a>.</p>
```
Comment (joejulian) [inline pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-new/index.md:60]:
```suggestion
   <p class="message--important"><strong>IMPORTANT: </strong>Not all released versions of Kubernetes are available in Azure when creating a Managed Cluster (AKS). Use the Azure portal or CLI to <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions">find the versions supported in your cluster's location</a>.</p>
```

Comment (cmastr) [reply pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-new/index.md:60]: Why can't we just tell them the versions?

Comment (dkoshkin) [reply pages/dkp/konvoy/2.1/choose-infrastructure/aks/aks-advanced/aa-new/index.md:60]: The versions may be updated at any time, we don't want to maintain the list ourselves.

**Review by dlipovetsky — APPROVED** [review]
Thank you!

**Pull request merged**